modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -51,6 +51,8 @@ const RichText = ({
 		}
 	}, [editingLanguageId, editorRef]);
 
+	const editorName = name.replaceAll('$', '_');
```

Comment:
@linolaoj, is there any chance of the `name` be `null` or `undefined`?

Please avoid creating a variable to be used only once as a parameter, instead use as parameter `name.replaceAll('$', '_')`. :wink:

Reply:
I'm going to include this change